CHANGELOG.md

```diff
@@ -20,6 +20,8 @@
 * Adds the ability to style the text in the inverted button with a spinner - yuki24
 * Now the `<ConfirmBid>` component shows a spinner while loading - yuki24
 * Now the `<Input>` component updates the border color when the parent component updates the `error` prop - yuki24
+* `dismissModalViewController` now returns a promise - ash&sarah
```

Comment:
Should we see native code for this? I don't see any

Reply:
Yup, a small change here: https://github.com/artsy/emission/pull/1065/files#diff-c389a83eac4316bce2520620bd807d14R42

Reply:
Ace, can you bump the [native-code-version](https://github.com/artsy/emission/blob/develop-1.5.x/package.json#L4) in the package.json - wonder what's up with the [danger rule](https://github.com/artsy/emission/blob/develop-1.5.x/dangerfile.ts#L184-L195)

Reply:
Fixed the rule here, next CI should fail: https://github.com/artsy/emission/pull/1065/commits/cb73d7e960b231beab6d00fdadb3eabdaed546ef